docs/mp/beanvalidation.adoc

```diff
@@ -191,6 +193,7 @@ Supported types are:
 |`@Size`
 a|The annotated element size must be between the specified boundaries (included).
 Supported types are:
+
```

Comment:
Without this empty line content looks a little wrong.
Example:
<img width="1097" alt="Screenshot 2024-04-18 at 18 49 37" src="https://github.com/helidon-io/helidon/assets/4740207/6cb3c5ff-831c-4b57-8fd6-1f785f86ac62">

With this empty line:
<img width="1080" alt="Screenshot 2024-04-18 at 18 50 46" src="https://github.com/helidon-io/helidon/assets/4740207/d0e5de25-91a2-4a0e-bb9a-d0fff51f1484">